host-configs/ruby-toss_3_x86_64_ib-intel@18.0.2.cmake

```diff
@@ -53,7 +53,7 @@ set(ENABLE_MPI ON CACHE BOOL "")
 # Hardware Specifics
 #------------------------------------------------
 
-set(ENABLE_OPENMP ON CACHE BOOL "")
+set(ENABLE_OPENMP OFF CACHE BOOL "")
```

Comment:
For my information, why the change here?  Was this part of the disk space problem fix, or was OpenMP compilation taking too long, or what?

Reply:
And why is OpenMP turned off in the Intel host-configs on ruby, but on rzgenie?

Reply:
@agcapps -- I disabled OpenMP in our intel builds due to excessive build times, and OpenMP coverage in other host-configs.  We also discussed this during a team meeting on 5/10/21.
Looks like I only got the RZ host-configs in that PR, and this picks up the CZ ones.

Disabling OpenMP helped significantly for the intel@19 configuration, which went down from around 30 minutes to around 5 minutes.  Unfortunately, the intel@18 configs still take around 30+ minutes.

See:
* https://github.com/LLNL/axom/issues/544
* https://github.com/LLNL/axom/pull/553

Reply:
Thanks for the clarification, @kennyweiss .